From 4d85bd9d7c3bbe176a0896c6c4d991d834456a75 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <michael.i.doherty@intel.com>
Date: Wed, 7 Sep 2016 14:49:02 +0100
Subject: [PATCH 1/4] Use Clear Linux stateless directories by default

This will ensure that both PAM and D-BUS work for LightDM out of the box
within Clear Linux

Signed-off-by: Ikey Doherty <michael.i.doherty@intel.com>
---
 data/Makefile.am | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/data/Makefile.am b/data/Makefile.am
index b3dd2f8..3751bbe 100644
--- a/data/Makefile.am
+++ b/data/Makefile.am
@@ -4,10 +4,10 @@ dist_init_DATA = init/lightdm.conf
 confdir = $(sysconfdir)/lightdm
 dist_conf_DATA = lightdm.conf users.conf keys.conf
 
-dbusconfdir = $(sysconfdir)/dbus-1/system.d
+dbusconfdir = $(datadir)/dbus-1/system.d
 dist_dbusconf_DATA = org.freedesktop.DisplayManager.conf
 
-pamdir = $(sysconfdir)/pam.d
+pamdir = $(datadir)/pam.d
 dist_pam_DATA = pam/lightdm \
                 pam/lightdm-autologin \
                 pam/lightdm-greeter
-- 
2.10.0

